shapely/tests/test_constructive.py

```diff
@@ -261,7 +261,7 @@ def test_offset_curve_distance_array():
 def test_offset_curve_kwargs():
     # check that kwargs are passed through
     result1 = shapely.offset_curve(
-        line_string, -2.0, quadsegs=2, join_style="mitre", mitre_limit=2.0
+        line_string, -2.0, quadsegs=2, join_style="MITRE", mitre_limit=2.0
```

Comment:
```suggestion
        line_string, -2.0, quadsegs=2, join_style="mitre", mitre_limit=2.0
```

(let's leave some as lower case, to ensure this is covered that it works)